### PR TITLE
tests, integration-tests: port the op remove retry test to spread

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: tests/main/op-remove-retry/task.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  echo "Given a snap is installed"
+  snapbuild $TESTSLIB/snaps/test-snapd-tools .
+  snap install test-snapd-tools_1.0_all.snap
+
+  echo "And its mount point is kept busy"
+  systemd-run --unit unmount-blocker $(which test-snapd-tools.block)
+  wait_for_service unmount-blocker active
+
+  echo "When we try to remove the snap"
+  snap remove test-snapd-tools &
```
Comment (zyga) [inline tests/main/op-remove-retry/task.yaml]: We should perhaps keep the pid of this job and kill it later

**Comment by zyga** [comment]
Just a small nitpick. The test looks good

**Comment by fgimenez** [comment]
@zyga hey, about killing the remove job, ideally the command should finish after the blocker service is stopped, I've added to the restore step a `kill %1` with a `|| true` guard for taking into account the happy path, pls have another look.

Thanks! :)

**Comment by zyga** [comment]
Looks good

**Comment by mvo5** [comment]
Looks good, branch has conflicts now though. Out of curiosity, how long does this test take to run?

**Comment by fgimenez** [comment]
thanks for your review @mvo5 ! :) deconflicted, the previous execution [1] took 2s in i386 and 3s in amd64, let's see how long takes this one.

[1] https://travis-ci.org/snapcore/snapd/jobs/141344086

**Comment by mvo5** [comment]
Nice!

**Pull request merged**